.github/workflows/integration-opensuse-leap.yml

```diff
@@ -1,14 +1,5 @@
 name: integration-opensuse-leap
-
-on:
-  push:
-    branches:
-      - openSUSE-3004
```

Comment:
Why is it "okay" to remove naming these branches? Doesn't this cause double-executions on PRs from this repository?